src/backend/utils/uninstaller.ts

```diff
@@ -0,0 +1,125 @@
+import { GlobalConfig } from 'backend/config'
+import { fixesPath, gamesConfigPath } from 'backend/constants'
+import { notify } from 'backend/dialog/dialog'
+import { logError, logInfo, LogPrefix } from 'backend/logger/logger'
+import { gameManagerMap } from 'backend/storeManagers'
+import { sendGameStatusUpdate } from 'backend/utils'
+import { Runner } from 'common/types'
+import { storeMap } from 'common/utils'
+import { Event } from 'electron'
+import { existsSync, readdirSync, rmSync } from 'graceful-fs'
+import i18next from 'i18next'
+import { join } from 'path'
+
+export const removePrefix = async (appName: string, runner: Runner) => {
+  const { winePrefix } = await gameManagerMap[runner].getSettings(appName)
+  logInfo(`Removing prefix ${winePrefix}`, LogPrefix.Backend)
+
+  if (!existsSync(winePrefix)) {
+    logInfo(`Prefix folder ${winePrefix} doesn't exist, ignoring removal`)
+    return
+  }
+
+  // folder exists, do some sanity checks before deleting it
+  const { defaultInstallPath, defaultWinePrefix } =
+    GlobalConfig.get().getSettings()
+
+  if (winePrefix === defaultInstallPath) {
+    logInfo(
+      `Can't delete folder ${winePrefix}, prefix folder is the default install directory ${defaultInstallPath}`
+    )
+    return
+  }
+
+  if (winePrefix === defaultWinePrefix) {
+    logInfo(
+      `Can't delete folder ${winePrefix}, prefix folder is the default prefix directory ${defaultWinePrefix}`
+    )
+    return
+  }
+
+  const dirContent = readdirSync(winePrefix)
+
+  if (dirContent.length > 0) {
+    const driveCPath = join(winePrefix, 'drive_c')
+
+    if (!existsSync(driveCPath)) {
```

Comment:
Note that Proton prefixes (not using umu) will never have this folder. Maybe we also want to check for a `pfx` folder?